retrieve_nearby_books.py

```diff
@@ -0,0 +1,176 @@
+#code to get json response for /v5/bibs/search
+import requests
+import base64
+import PIL
+import io
+from PIL import Image, ImageDraw, ImageFont
+from collections import defaultdict
+import textwrap
+from IPython.display import HTML
+from io import BytesIO
+import struct
+
+"""
+There are two steps for getting a response using sierra API
+        1) Authenticating with the Patron API
+        2) Making a Sierra API call
+"""
+def load_client_key():
+    #Step1
+    #Loading Client key and secret key from the file
+    f=open('C:\\uc\\uclib\\credentials.txt', 'r')
+    lines = f.readlines()
+
+    user=lines[0].replace('\n','')
+    pwd=lines[1].replace('\n','')
+    
+    #Combining the API key and secret into a single string separated by a colon:
+    comb = user+':'+pwd
+    return comb
+
+def generate_encoded_key(combined_clientkey_secretkey):
+    #Converting the combined key:secret string into a Base64 string
+    comb_enc = base64.b64encode(bytes(combined_clientkey_secretkey, 'utf-8')).decode("utf-8")
+    return comb_enc
+
+def authorize_api(combined_key_encrypted):
+    #Setting the Header name field to authorization.
+    #Setting the Header value field to Basic, and inserting the Base64-encoded credentials
+    headers = {
+        'authorization': "Basic "+combined_key_encrypted,
+        'content-type': "application/json",
+    }
+
+    #Set the method to POST
+    r_step1 = requests.post('https://uclid.uc.edu:443/iii/sierra-api/v5/token', headers=headers)
+    data = r_step1.json()
```

Comment:
Can we combine into just one line of code by using .json() after .post like this?
`data = requests.post('https://uclid.uc.edu:443/iii/sierra-api/v5/token', headers=headers).json()`